ui/rtk-query/ability.js

```diff
@@ -60,5 +53,9 @@ export const LoadSessionGuard = ({ children }) => {
     );
   }
 
-  return children;
+  return (
+    <LoadingScreen isLoading={isLoading || !org?.id} message="Setting up your session">
```

Comment:
@leecalcote  you might want to change the text here . this is show during loading org , setting up keys

Reply:
Nice, do we have an array of possibilities?

Reply:
not right now , but we could have similar setup . i didnt make it random choice so users know at what stage they are in . 
from loading app -> setting up session -> loading canvas .